public/index.html

```diff
@@ -15,6 +15,13 @@
 </style>
 <html>
   <body>
+    <h2>Current status</h2>
+    <h3>
+      No. of nodes: <span id="nodes_number"></span><br>
+      Last update: <span id="last_update"></span><br>
+      Next refresh: <span id="refresh_interval"></span><br>
+      Requests used: <span id="requests_used"></span><br>
+    </h3>
     <svg width="1280" height="720"></svg>
```

Comment:
I don't understand, could you explain what the issue is? (I haven't changed anything in the latest commits, so your comment should still be relevant)